lib/blemulator/internal/simulation_manager_mixins/characteristics_mixin.dart

```diff
@@ -1,6 +1,8 @@
 part of internal;
 
 mixin CharacteristicsMixin on SimulationManagerBase {
+  Map<String, StreamSubscription> _monitoringSubscriptions = HashMap();
```

Comment:
This should keep all transactions connected to specified subscriptions. What's MBA's behaviour if you try to monitor same characteristic twice? Consult @Cierpliwy

Reply:
Multiple monitoring of the same characteristic is allowed in real BT stack